{
	"DENY": {
		"TARGET":  {"type": "cybox:Network_Connection", 
			"cybox:Object": {
				"cybox:Properties": {
					"xsi:type": "NetworkConnectionObj:NetworkConnectionObjectType",
					"NetworkConnectionObj:Layer3_Protocol": "IPv4",
					"NetworkConnectionObj:Layer4_Protocol": "TCP",
					"NetworkConnectionObj:Source_Socket_Address": {
						"SocketAddressObj:IP_Address": {
							"AddressObj:Address_Value": "any"
						}
					},
					"NetworkConnectionObj:Destination_Socket_Address": {
						"SocketAddressObj:IP_Address": {
							"AddressObj:Address_Value": "10.10.10.2"
						}
					}
				}
			}
		  }, 
		"ACTUATOR": {"type": "network.firewall", 
			"id": "30",
			"connector_name": "Cisco ASA",
			"connector_run_id": 136,
			"config": {
			   "app_version": "1.2.9",
			   "appname": "-",
			   "device": "10.18.2.2",
			   "directory": "ciscoasa_0409C19F-FEE2-4D22-879B-96D90619BD6C",
			   "enable_password": "",
			   "main_module": "ciscoasa_connector.pyc",
			   "password": "YTjleuFwkPU8CtrYWF2BEQ==",
			   "username": "pix"
			}
		},
		"MODIFIERS": [
			{"context_ref": 91},
			{"debug_level": 3},
			{"identifier": "block_ip"}
			{"container_label": "incident"},
			{"container_name": "Incident # 1"},
			{"container_severity": "medium"},
			{"ipc_version": 1}
			{"phantom_version": "1.2.265"},
			{"session_id": "822e7767-25ac-49ed-8d21-115840cb6aa9"}
		]
	}
}
